infra/events/src/test/java/com/simprints/infra/events/event/local/migrations/EventMigration10to11Test.kt

```diff
@@ -119,8 +119,8 @@ class EventMigration10to11Test {
                 "id":"$id",
                 "projectId":"$PROJECT_ID",
                 "type":"$OLD_CAPTURE_SESSION_EVENT_KEY",
-                "createdAt":$CREATED_AT,
-                "endedAt":${ENDED_AT.takeIf { ended } ?: 0},
+                "startTime":$CREATED_AT,
```

Comment:
Changing historic migration tests feels wrong. It should ensure that classes in that specifc state are migrated correctly, there will be no new fields in the real-life migration